src/ansible_navigator/actions/__init__.py

```diff
@@ -1,4 +1,17 @@
-"""actions for Explorer"""
+"""A package containing all available actions.
+
+The actions package is a plugin like implementation for all available application actions.
+
+Actions can be added without registration and are discovered and loaded
+when their ``KEGEX`` is matched against user input.
+
+This allows for each action to be loaded multiple times, creating a unique
+and isolated instance of the action.
+
+Currently, the actions package is the only package supported for actions and
```

Comment:
Should we highlight package names?
```suggestion
Currently, the ``actions`` package is the only package supported for actions and
```

Reply:
why not!